data/src/main/scala/ch.epfl.scala.index.data/bintray/SbtPluginsData.scala

```diff
@@ -59,3 +50,46 @@ case class SbtPluginReleaseModel(
     created: String,
     sha1: String
 )
+
+object SbtPluginReleaseModel {
+  def apply(
+      subject: String,
+      repo: String,
+      organization: String,
+      artifact: String,
+      scalaVersion: String,
+      sbtVersion: String,
+      version: String,
+      ivyPath: String,
+      sha1: String,
+      descriptor: ModuleDescriptor
+  ): SbtPluginReleaseModel = {
+    val releaseModel = ReleaseModel(
+      groupId = organization,
+      artifactId = artifact,
+      version = version,
+      packaging = "jar",
+      description = Option(descriptor.getDescription),
+      dependencies = descriptor.getDependencies.toList.map { dependency =>
+        val dep = dependency.getDependencyRevisionId
+        Dependency(
+          dep.getOrganisation,
+          dep.getName,
+          dep.getRevision,
+          scope = /* TODO */ None,
```

Comment:
Is this a change with the previous behavior?

Reply:
No it does not. It is code that I have moved to this place for readability.